src/test/java/com/jcabi/github/RtStatusesTest.java

```diff
@@ -0,0 +1,130 @@
+ */
+public class RtStatusesTest {
+
+    /**
+     * Tests creating a Commit.
+     *
+     * @throws Exception when an error occurs
+     */
+    @Test
+    public final void createsStatus() throws Exception {
+        final String sha = "0abcd89jcabitest";
+        final MkContainer container = new MkGrizzlyContainer().next(
+            new MkAnswer.Simple(
+                HttpURLConnection.HTTP_CREATED,
+                "{\n" +
+                        "  \"created_at\": \"2012-07-20T01:19:13Z\",\n" +
+                        "  \"updated_at\": \"2012-07-20T01:19:13Z\",\n" +
+                        "  \"state\": \"failure\",\n" +
+                        "  \"target_url\": \"https://ci.example.com/1000/output\",\n" +
+                        "  \"description\": \"Build has completed successfully\",\n" +
+                        "  \"id\": 1,\n" +
+                        "  \"url\": \"https://api.github.com/repos/octocat/Hello-World/statuses/1\",\n" +
+                        "  \"context\": \"continuous-integration/jenkins\",\n" +
+                        "  \"creator\": {\n" +
+                        "    \"login\": \"octocat\",\n" +
+                        "    \"id\": 1,\n" +
+                        "    \"avatar_url\": \"https://github.com/images/error/octocat_happy.gif\",\n" +
+                        "    \"gravatar_id\": \"\",\n" +
+                        "    \"url\": \"https://api.github.com/users/octocat\",\n" +
+                        "    \"html_url\": \"https://github.com/octocat\",\n" +
+                        "    \"followers_url\": \"https://api.github.com/users/octocat/followers\",\n" +
+                        "    \"following_url\": \"https://api.github.com/users/octocat/following{/other_user}\",\n" +
+                        "    \"gists_url\": \"https://api.github.com/users/octocat/gists{/gist_id}\",\n" +
+                        "    \"starred_url\": \"https://api.github.com/users/octocat/starred{/owner}{/repo}\",\n" +
+                        "    \"subscriptions_url\": \"https://api.github.com/users/octocat/subscriptions\",\n" +
+                        "    \"organizations_url\": \"https://api.github.com/users/octocat/orgs\",\n" +
+                        "    \"repos_url\": \"https://api.github.com/users/octocat/repos\",\n" +
+                        "    \"events_url\": \"https://api.github.com/users/octocat/events{/privacy}\",\n" +
+                        "    \"received_events_url\": \"https://api.github.com/users/octocat/received_events\",\n" +
+                        "    \"type\": \"User\",\n" +
+                        "    \"site_admin\": false\n" +
+                        "  }\n" +
+                        "}"
+            )
+        ).start();
+        final Request req = new ApacheRequest(container.home());
+        final Statuses statuses = new RtStatuses(req, new RtCommit(req, repo(), sha));
+
+        try {
+            final Status newStatus = statuses.create(
```

Comment:
You mean for the variable or class -> Should I migrate `Statuses` to `Status`? 'Cause if 'yes', than there are two similar classes: `Statuses` and `Status` already in the code.